asset/src/kafka_reader/interfaces.ts

```diff
@@ -60,4 +60,14 @@ export interface KafkaReaderConfig extends OpConfig {
      * assigns partitions to group members.
     */
     partition_assignment_strategy?: 'range'|'roundrobin';
+    /**
+     * Name of kafka api used for reader, if none is provided, then one is made
+     * and the name is kafka_reader_api, and is injected into the execution
+    */
+    api_name?: string;
+    /**
+     * How to decode data from buffer, defaults to JSON
+     *
+    */
+    _encoding?: DataEncoding
```

Comment:
This is automatically added to to all operation schemas in `job-components` - we don't need to specify this twice.